independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeInterceptorDeployment.java

```diff
@@ -152,6 +153,9 @@ private <T> LinkedHashMap<ResourceInterceptor<T>, T> createInterceptorInstances(
         List<BeanFactory.BeanInstance<T>> responseBeanInstances = new ArrayList<>(interceptors.size());
         Collections.sort(interceptors);
         for (ResourceInterceptor<T> interceptor : interceptors) {
+            if (RuntimeType.CLIENT.equals(interceptor.getRuntimeType())) {
+                continue;
+            }
```

Comment:
I see you fix it for the server. Is it already handled in the client (i.e. interceptors that are server only are not run for the client?)?

Reply:
Good point. Let me checkl

Reply:
I checked and the client already behaves as expected